test/nyc-bin.js

```diff
@@ -48,6 +48,38 @@ describe('the nyc cli', function () {
     })
   })
 
+  describe('report and check', function () {
+    it('should show coverage check along with report', function (done) {
```

Comment:
Maybe we should have a second test to verify behavior when `--check-coverage` is not specified?  Granted we should have already had that but just an observation.

Reply:
Actually I can take care of this at a later time.  Unless you're already working on this don't worry about it.